### PR TITLE
20392 Update completed IA/amalg/reg components

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#20392

*Description of changes:*
- app version = 7.1.8
- renamed "draft app" objects to "bootstrap filing"
- store bootstrap filing identifier
- updated Amalgamation filing component
- updated Incorporation Application filing component
- updated Registration filing component
- added a store getter
- updated unit tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-641-v2utk9f2.web.app

File: src/App.vue [hunk 1]
```diff
@@ -457,10 +457,21 @@ export default class App extends Mixins(
...
+        // fetch the bootstrap filing and store the bootstrap business data
+        const response = await LegalServices.fetchBootstrapFiling(this.tempRegNumber)
+        this.storeBootstrapBusinessData(response)
+
+        // if it is a task (not a filing), and it has a NR, load it
+        if (this.isBootstrapTask && this.localNrNumber) {
+          const nr = await LegalServices.fetchNameRequest(this.localNrNumber)
+          this.storeNrData(nr, response)
+        }
+
```
Comment (severinbeauvais) [inline src/App.vue:474]: I moved a function's body here.

File: src/App.vue [hunk 2]
```diff
@@ -732,15 +727,15 @@ export default class App extends Mixins(
     // add this as a filing item
     const filingItem = {
       availableOnPaperOnly: header.availableOnPaperOnly,
-      businessIdentifier: this.getIdentifier,
-      commentsCount: application.commentsCount,
-      commentsLink: application.commentsLink,
-      displayLedger: application.displayLedger,
+      businessIdentifier: filing.business.identifier || this.getIdentifier,
```
Comment (severinbeauvais) [inline src/App.vue:730]: This is key -- save the new business id (eg, BCxxx) if it's present (which it is if the filing is completed), else fall back to the current idenfier (eg, Txxx).

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-641-v2utk9f2.web.app

**Review by jamespaologarcia — APPROVED** [review]
LGTM

**Review by ketaki-deodhar — APPROVED** [review]
Looks good!

**Pull request merged**